indico/modules/rb_new/operations/rooms.py

```diff
@@ -90,6 +94,39 @@ def get_managed_room_ids(user):
         return {r.id for r in Room.get_owned_by(user)}
 
 
+# @memoize_redis(3600)
+def get_room_statistics(room):
+    data = {
+        'count': {
+            'title': 'Times booked',
```

Comment:
i18n.. best to just use an identifier here (e.g. `times_booked`) and then keep the human-friendly display on the client side

Reply:
You mean having a mapping const in the client like this?
```js
const mapping = {
    times_booked: 'Times booked',
    occupancy: 'Occupancy',
}
```

Reply:
yes (but with i18n ofc)